projects/netgrif-components-core/src/lib/navigation/model/group-navigation-constants.ts

```diff
@@ -43,12 +43,12 @@ export enum GroupNavigationConstants {
     /**
      * EnumerationMap field, that contains selected search type for case view
      * */
-    ITEM_FIELD_ID_CASE_VIEW_SEARCH_TYPE = 'case_view_search_type',
+    ITEM_FIELD_ID_CASE_VIEW_SEARCH_TYPE = 'view_search_type',
```

Comment:
shouldn't ITEM_FIELD_ID_CASE_VIEW_SEARCH_TYPE and ITEM_FIELD_ID_TASK_VIEW_SEARCH_TYPE be merged together? one of them removed